pages/flow/index.js

```diff
@@ -71,19 +73,26 @@ export default function Flow() {
                 onClick={() => {
                   setOpen(false);
                   setSearchQuery("");
+                  setFilterQuery("all");
                 }}
               >
                 X
               </StyledCloseButton>
             </SectionHeader>
             <SearchBar setSearchQuery={setSearchQuery} />
+            <LevelFilter setFilterQuery={setFilterQuery} />
             <StyledList>
               {asanas
                 .filter((asana) => {
                   const nameInLowerCase = asana.english_name.toLowerCase();
                   const searchQueryInLowerCase = searchQuery.toLowerCase();
                   return nameInLowerCase.includes(searchQueryInLowerCase);
                 })
+                .filter((asana) => {
+                  if (filterQuery !== "all") {
+                    return asana.levels[0] === filterQuery;
```

Comment:
I think you could use a ternary here if you want

Reply:
Hi Philipp, I just tried it and it didn't work well. I think probably because there are two conditions (filterQuery should not be all and the first element of the array should match the filterQuery)?

Reply:
Yes, this could be the case here. Sorry for the confusing comment :/

Reply:
No problem :) Always happy about feedback :)